install/helm/README.md

```diff
@@ -160,8 +160,9 @@ The following table lists the configurable parameters of the Thunder chart and t
 | `configuration.gateClient.hostname`    | Gate client hostname                                            | `0.0.0.0`                    |
 | `configuration.gateClient.port`        | Gate client port                                                | `8090`                       |
 | `configuration.gateClient.scheme`      | Gate client scheme                                              | `https`                      |
-| `configuration.gateClient.loginPath`   | Gate client login path                                          | `/gate/signin`                     |
-| `configuration.gateClient.errorPath`   | Gate client error path                                          | `/error`                     |
+| `configuration.gateClient.path`        | Gate client base path                                           | `/gate`                      |
+| `configuration.gateClient.loginPath`   | Gate client login path                                          | `/gate/signin`               |
```

Comment:
Shouldn't the loginPath and errorPath default values needs to be changed?
Or change the representation to say that they need to provider either path or (loginPath and errorPath)

Reply:
Removed loginPath & errorPath with
https://github.com/asgardeo/thunder/compare/ab1228a1d827ac3393b8f18fd1a2fc4ee5471cf6..ac42ff227c64f0a432e688ce7b4e9e7ba174626f